packages/android_alarm_manager/android/build.gradle

```diff
@@ -1,3 +1,14 @@
+def PLUGIN = "android_alarm_manager";
+def ANDROIDX_WARNING = "flutterPluginsAndroidXWarning";
+if (!rootProject.ext.has(ANDROIDX_WARNING)) {
```

Comment:
Discussed offline. I'm still looking into if there's something better to do here, but unfortunately I'm not sure if this is feasible since the build can break based on incompatible libraries included in other gradle files in addition to the root project's compileSdkVersion.